pipelined/bevy_sprite2/src/render/mod.rs

```diff
@@ -142,6 +145,141 @@ impl FromWorld for SpriteShaders {
             pipeline,
             view_layout,
             material_layout,
+            marker: PhantomData,
+        }
+    }
+}
+
+impl FromWorld for SpriteShaders<Sprite3d> {
```

Comment:
Just leaving a note to say that I haven't looked at this as part of my review of your PR, because I don't understand these rendering APIs well enough to review it.

Reply:
The SpriteShaders are basically the same as before. The difference is that the pipeline for 3d needs a depth attachment which the 2d one didnt have (because no depth in 2d).

There is room for generalization/code recycling here. But I dont think this is worth the effort atm. This is only a basic implementation making sprite available in 3d. I haven't added any feature which are possible in 3d now (e.g. normals) so the pipelines might diverge more with future additions. Just to reason about my decision here.